app/code/Magento/Config/README.md

```diff
@@ -1,8 +1,8 @@
-#Config
+# Magento_Config module
+
+## Config
```

Comment:
Remove the heading as this is a duplicate of the level 1 heading.

Reply:
**done**